docs/whatsnew/0.6.0.rst

```diff
@@ -91,7 +91,7 @@ Features
 - For `plasmapy.plasma.grids` functionality, improve interpolation performance on
   non-uniform grids. (`#963 <https://github.com/plasmapy/plasmapy/pull/963>`__)
 - Added the `~plasmapy.formulary.drifts.diamagnetic_drift` function to
-  `~plasmapy.formulary.drifts`. (`#966 <https://github.com/plasmapy/plasmapy/pull/966>`__)
+  `~plasmapy.formulary.drifts`. (`#972 <https://github.com/plasmapy/plasmapy/pull/972>`__)
```

Comment:
It turns out that #966 was the issue being addressed, and #972 was the pull request that addressed it.